src/layout/Header/index.tsx

```diff
@@ -115,33 +108,12 @@ export default function Header() {
               </NetworkCard>
             )}
           </Hidden>
-          {aggregateBalance && (
-            <PNGWrapper onClick={() => setShowPngBalanceModal(true)}>
-              <PNGAmount active={!!account} style={{ pointerEvents: 'auto' }}>
-                {account && (
-                  <Hidden upToSmall>
-                    <Text
-                      color="black"
-                      style={{
-                        paddingRight: '.4rem'
-                      }}
-                    >
-                      <CountUp
-                        key={countUpValue}
-                        isCounting
-                        start={parseFloat(countUpValuePrevious)}
-                        end={parseFloat(countUpValue)}
-                        thousandsSeparator={','}
-                        duration={1}
-                      />
-                    </Text>
-                  </Hidden>
-                )}
-                PNG
-              </PNGAmount>
-              <CardNoise />
-            </PNGWrapper>
-          )}
+          <PNGWrapper onClick={() => setShowPngBalanceModal(true)}>
```

Comment:
on 67th line number we should wrap `<PngBalanceContent setShowPngBalanceModal={setShowPngBalanceModal} />` with condition like this
```ts
{showPngBalanceModal && <PngBalanceContent setShowPngBalanceModal={setShowPngBalanceModal} />}
```
so that PngBalanceContent only load if modal is open